core/test/Governor.js

```diff
@@ -47,9 +50,10 @@ contract("Governor", function(accounts) {
     // Set the inflation rate to 0 by default, so the balances stay fixed.
     await setNewInflationRate("0");
 
-    // To work, the governor must be the owner of the Voting contract. This is not the default setup in the test
+    // To work, the governor must be the owner of the Voting and IdentifierWhitelist contracts. This is not the default setup in the test
```

Comment:
Will this test still work if we remove the `await voting.transferOwnership(governor.address);` line? IIUC, it shouldn't be necessary now that the identifier whitelist is a separate contract.

Reply:
Good catch, Governor no longer needs to be the `owner` of Voting